src/OnboardingSPA/pages/Steps/DesignColors/index.js

```diff
@@ -84,6 +84,13 @@ const StepDesignColors = () => {
 					</div>
 				</div>
 				<div className="theme-colors-preview__live-preview-container">
+					{!pattern && (
+						<LivePreview
+							blockGrammer={''}
+							styling={'custom'}
+							viewportWidth={1300}
+						/>
+					)}
```

Comment:
Again we can combine these using `?`

Reply:
It doesn't work here cause we are listening to pattern and not GlobalStyles